frontend/src/layouts/BaseLayout.vue

```diff
@@ -20,15 +20,15 @@
                     <span class="primary header-black q-ml-md">Umbra</span>
                   </div>
                   <div v-else-if="$q.screen.xs">
+                    <div v-if="isLoading" class="q-ml-md row">
+                      <q-spinner color="primary" size="1em" />
+                    </div>
```

Comment:
My localStorage has `__q_strn|MetaMask` as the value for the `last-wallet` key, but when testing out this PR it gets stuck with `isLoading = true`, and there's a few (possibly related) errors in the console. Some screenshots below (might need to open the image in a new tab to see the full resolution)

![image](https://user-images.githubusercontent.com/17163988/132781797-af216e9a-aadc-4050-bc4b-9ee80855eac0.png)

![image](https://user-images.githubusercontent.com/17163988/132781818-e1e9cfc7-b707-4119-81ed-89d06f98865d.png)

![image](https://user-images.githubusercontent.com/17163988/132781866-bf6b62af-6a11-497d-afbe-21f682db7646.png)

Reply:
What exactly did you do?

Reply:
I just built the app locally like normal. Also just noticed it seems my screenshot of the header was poorly timed as its from the small portion of the spinner cycle when there is no spinner 🙃

> Try clearing site data.

I thought about this, but didn't want to do it, and don't think we should merge this PR if it requires clearing site data—the reason is because if you need to clear site data for it to work properly, then the live app will break for users once deployed and we don't want to make them clear their data

Reply:
Yep, you're right. I think I got it now.

Reply:
Yes, seems to be working now, thank you!